web-core-sdk/one-key-flow/src/App.tsx

```diff
@@ -0,0 +1,360 @@
+    null
+  );
+
+  useEffect(() => {
+    const init = async () => {
+      try {
+        // Initialising Web3Auth
+        const web3auth = new Web3AuthCore({
+          clientId,
+          chainConfig: {
+            chainNamespace: CHAIN_NAMESPACES.EIP155,
+            ...chainConfig,
+          },
+        });
+
+        const openloginAdapter = new OpenloginAdapter({
+          loginSettings: {
+            mfaLevel: "default",
+          },
+          adapterSettings: {
+            network: "testnet",
+            uxMode: "redirect",
+            loginConfig: {
+              jwt: {
+                name: "Web3Auth One Key Login Flow",
+                verifier,
+                typeOfLogin: "jwt",
+                clientId,
+              },
+            },
+          },
+        });
+        web3auth.configureAdapter(openloginAdapter);
+        setWeb3auth(web3auth);
+        await web3auth.init();
+
+        // Instantiating Torus for One Key Flow
+        const torus = new Torus({
+          enableOneKey: true,
+          network,
+        });
+        setTorus(torus);
+
+        const nodeDetailManager = new NodeDetailManager({ network, proxyAddress: CONTRACT_MAP[TORUS_NETWORK.TESTNET] });
+        setNodeDetailManager(nodeDetailManager);
+
+        if (web3auth.provider) {
+          setProvider(web3auth.provider);
+        }
+      } catch (error) {
+        console.error(error);
+      }
+    };
+
+    init();
+  }, []);
+
+  const signInWithGoogle = async (): Promise<UserCredential> => {
+    try {
+      const app = initializeApp(firebaseConfig);
+      const auth = getAuth(app);
+      const googleProvider = new GoogleAuthProvider();
+      const res = await signInWithPopup(auth, googleProvider);
+      console.log(res);
+      return res;
+    } catch (err) {
+      console.error(err);
+      throw err;
+    }
+  };
+
+  const login = async () => {
+    if (!web3auth) {
+      uiConsole("web3auth not initialized yet");
+      return;
+    }
+    // login with firebase
+    const loginRes = await signInWithGoogle();
+    // get the id token from firebase
+    const idToken = await loginRes.user.getIdToken(true);
+
+    // parse the id token to get the sub value - this will be verifier id (used to know the user)
+    const base64Url = idToken.split('.')[1]
+    const base64 = base64Url?.replace('-', '+').replace('_', '/')
+    const { sub } = JSON.parse(window.atob(base64 || ''))
+
+    // get details of the node shares on the torus network
+    const { torusNodeEndpoints, torusNodePub, torusIndexes } = await nodeDetailManager.getNodeDetails({ verifier, verifierId: sub });
+    const userDetails = await torus.getUserTypeAndAddress(torusNodeEndpoints, torusNodePub,  { verifier, verifierId: sub }, true);
+
+    // check if the user hasn't enabled one key login
+    if (userDetails.typeOfUser === "v2" && (userDetails.nonce === 0 || !userDetails.nonce)) {
```

Comment:
this check if incorrect. Plz check my code snippet , it should return false here is nonce eq to 0 or void.

Reply:
I changed me code according to your example.